### PR TITLE
normal team user show

### PR DESCRIPTION
**Pull request by minhld1709**

### CODE REVIEW

File: app/views/teams/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                <td><%= user.birthday %></td>
+              </tr>
+            <% end %>                
+          </tbody>        
+        </table>        
+      </section>
+    </aside>
+  </div>
+</div>
+<% if current_user.id == @team.leader && !current_user.nil? %>
```
Comment (khanhkhanh) [inline app/views/teams/show.html.erb]: Try logging out, and this line may cause errors :)

File: config/locales/en.yml [hunk 2]
```diff
@@ -94,3 +94,5 @@ en:
   user_no_team: "List user no team"
   user_this_team: "List user this team"
   teams_company_banner: "All team in My company"
```
Comment (khanhkhanh) [inline config/locales/en.yml]: All teams in my company

File: config/locales/en.yml [hunk 3]
```diff
@@ -94,3 +94,5 @@ en:
   user_no_team: "List user no team"
   user_this_team: "List user this team"
```
Comment (khanhkhanh) [inline config/locales/en.yml]: List of non-team users

File: config/locales/en.yml [hunk 4]
```diff
@@ -94,3 +94,5 @@ en:
   user_no_team: "List user no team"
```
Comment (khanhkhanh) [inline config/locales/en.yml]: List of non-team users

File: app/controllers/teams_controller.rb [hunk 5]
```diff
@@ -2,4 +2,13 @@ class TeamsController < ApplicationController
   def index
     @teams = Team.paginate page: params[:page], per_page: Settings.page.per_max
```
Comment (khanhkhanh) [inline app/controllers/teams_controller.rb]: Change per_max to another name to make it clearer

File: app/views/teams/show.html.erb [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        </h2>
+        <table class="table">
+          <thead>
+            <tr>
+              <th><%= t "user.name" %></th>
+              <th><%= t "user.birthday" %></th>
+            </tr>
+          </thead>
+          <tbody>
+            <% @team.users.each do |user| %>
```
Comment (khanhkhanh) [inline app/views/teams/show.html.erb]: Create another view with name "_user.html.erb", then use render partial :)

File: app/views/teams/show.html.erb [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+          </tbody>        
+        </table>        
+      </section>
+    </aside>
+  </div>
+</div>
+<% if current_user.id == @team.leader && !current_user.nil? %>
+  <%= link_to t("edit.team"), edit_team_path(@team) %><br/>
+<% end %>
+<%= link_to t("back"),teams_path, class: "glyphicon glyphicon-step-backward" %>
```
Comment (khanhkhanh) [inline app/views/teams/show.html.erb]: Need a space before teams_path

File: app/views/teams/show.html.erb (hunk 1 above)
Comment (minhld1709) [inline app/views/teams/show.html.erb]: mình không hiểu rõ đoạn nè lắm

Comment (DucNguyenVan) [inline app/views/teams/show.html.erb]: Ý là phải check user.signed_in?

File: app/controllers/teams_controller.rb [hunk 8]
```diff
@@ -1,5 +1,14 @@
...
+    @teams = Team.paginate page: params[:page], per_page: Settings.page.max_page
+  end
+
+  def show
+    @team = Team.find params[:id]
+  end
+
+  private
+  def team_params
+    params.require(:team).permit :name, :description, :leader
```
Comment (dieunb) [inline app/controllers/teams_controller.rb]: `leader` or `leader_id`

File: app/views/admin/teams/show.html.erb [hunk 9]
```diff
@@ -1,10 +1,6 @@
 <% provide :title, @team.name %>
 <div class="container">
-  <h1>
-    <%= t("welcome_to")%>
```
Comment (brokenheart1712) [inline app/views/admin/teams/show.html.erb]: t :welcome_to

File: app/views/admin/teams/show.html.erb [hunk 10]
```diff
@@ -1,10 +1,6 @@
...
-    <%= @team.name %> 
-    <%= t("team")%>
```
Comment (brokenheart1712) [inline app/views/admin/teams/show.html.erb]: t :team

File: app/views/admin/teams/show.html.erb [hunk 11]
```diff
@@ -1,10 +1,6 @@
...
-  </h1>           
+  <h1><%= t "welcome_banner", name: @team.name %></h1>
```
Comment (brokenheart1712) [inline app/views/admin/teams/show.html.erb]: t :welcome_banner

File: app/views/teams/show.html.erb [hunk 12]
```diff
@@ -0,0 +1,29 @@
+<% provide :title, @team.name %>
+<div class="container">
+  <h1><%= t "welcome_banner", name: @team.name %></h1>
+  <div class="row">
+    <aside class="col-md-6 col-md-offset-3">
+      <section class="team_info">
+        <h2>
+          <%= t "description" %>:
```
Comment (brokenheart1712) [inline app/views/teams/show.html.erb]: t :description

File: app/views/teams/show.html.erb [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+          <tbody>
+            <%= render @team.users %>                
+          </tbody>        
+        </table>        
+      </section>
+    </aside>
+  </div>
+</div>
+<% if !current_user.nil? && current_user.id == @team.leader_id %>
+  <%= link_to t("edit.team"), edit_team_path(@team) %><br/>
```
Comment (brokenheart1712) [inline app/views/teams/show.html.erb]: t(:edit_team)

File: app/views/teams/show.html.erb [hunk 14]
```diff
@@ -0,0 +1,29 @@
...
+<% end %>
+<%= link_to t("back"), teams_path, class: "glyphicon glyphicon-step-backward" %>
```
Comment (brokenheart1712) [inline app/views/teams/show.html.erb]: t(:back)

**Pull request merged**